.github/workflows/main.yml

```diff
@@ -0,0 +1,116 @@
+  CIBW_BUILD: "cp36-* cp37-* cp38-*"
+  CIBW_BEFORE_BUILD: "pip install numpy==1.17.3 cython>=0.29.14 setuptools"
+  CIBW_TEST_REQUIRES: "pytest"
+  CIBW_TEST_COMMAND: "pytest -v {project}/tests"
+
+  # Allows you to run this workflow manually from the Actions tab
+  workflow_dispatch:
+
+jobs:
+  build_wheels:
+    name: wheels on ${{matrix.os}}
+    runs-on: ${{matrix.os}}
+    # only run on internal push's or external PRs, not on internal PRs
+    if: github.event_name == 'push' || github.event.pull_request.head.repo.full_name != github.repository
+    strategy:
+      fail-fast: false
+      matrix:
+        os: [ubuntu-latest, windows-latest, macos-latest]
+
+    steps:
+      - uses: actions/checkout@v2
+
+      - name: Set up Python
+        uses: actions\setup-python@v2
+        with:
+          python-version: "3.8"
+
+      - name: Install dependencies
+        run: |
+          python -m pip install --upgrade pip
+          pip install cibuildwheel>=1.7.0
+      - name: Build wheels
+        run: |
+          python -m cibuildwheel --output-dir dist
+
+      - uses: actions/upload-artifact@v2
+        with:
+          path: ./dist/*.whl
+
+  build_sdist:
+    name: sdist on ${{matrix.os}} with py ${{ matrix.python-version }}
+    runs-on: ${{matrix.os}}
+    # only run on internal push's or external PRs, not on internal PRs
+    if: github.event_name == 'push' || github.event.pull_request.head.repo.full_name != github.repository
+    strategy:
+      fail-fast: false
+      matrix:
+        os: [ubuntu-latest, windows-latest, macos-latest]
+        python-version: [3.6, 3.7, 3.8]
+
+    steps:
+      - uses: actions/checkout@v2
+
+      - name: Set up Python ${{ matrix.python-version }}
+        uses: actions\setup-python@v2
+        with:
+          python-version: ${{ matrix.python-version }}
+
+      - name: Install dependencies
+        run: |
+          python -m pip install --upgrade pip
+          pip install -r requirements_setup.txt
+          pip install -r requirements.txt
+          pip install -r requirements_test.txt
+
+      - name: Build sdist
+        run: |
+          python setup.py sdist -d dist
+          python setup.py --openmp build_ext --inplace
+
+      - name: Run tests
+        run: |
+          python -m pytest --cov gstools --cov-report term-missing -v tests/
+          python -m coveralls
+
+      - uses: actions/upload-artifact@v2
+        with:
+          path: dist/*.tar.gz
+
+  upload_to_pypi:
+    needs: [build_wheels, build_sdist]
+    runs-on: ubuntu-latest
+    # only run on internal push's or external PRs, not on internal PRs
+    if: github.event_name == 'push' || github.event.pull_request.head.repo.full_name != github.repository
+
+    steps:
+      - uses: actions/download-artifact@v2
+        with:
+          name: artifact
+          path: dist
+
+      - name: Publish to Test PyPI
+        # only if working on develop
+        # TODO change from master to develop
```

Comment:
Can you resolve this TODO?

Reply:
Ups... that got lost during a rebase session I guess, thank!